File: Algorithms/src/2833. Furthest Point From Origin/Code/solution.js
/**
 * @param {string} moves
 * @return {number}
 */
var furthestDistanceFromOrigin = function (moves) {
  let left = 0,
    right = 0,
    blank = 0;

  for (let c of moves) {
    if (c === "L") left++;
    else if (c === "R") right++;
    else blank++;
  }

  let position = right - left;
  return Math.abs(position) + blank;
};

File: Algorithms/src/2833. Furthest Point From Origin/Code/solution.go
func furthestDistanceFromOrigin(moves string) int {
    left, right, blank := 0, 0, 0

    for _, c := range moves {
        if c == 'L' {
            left++
        } else if c == 'R' {
            right++
        } else {
            blank++
        }
    }

    position := right - left

    if position < 0 {
        position = -position
    }

    return position + blank
}